src/main/java/uk/gov/companieshouse/acsp/manage/users/interceptor/LoggingInterceptor.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.companieshouse.acsp.manage.users.interceptor;
+
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import org.springframework.lang.NonNull;
+import org.springframework.lang.Nullable;
+import org.springframework.stereotype.Component;
+import org.springframework.web.servlet.HandlerInterceptor;
+import org.springframework.web.servlet.ModelAndView;
+import uk.gov.companieshouse.acsp.manage.users.utils.StaticPropertyUtil;
+import uk.gov.companieshouse.logging.Logger;
+import uk.gov.companieshouse.logging.LoggerFactory;
+import uk.gov.companieshouse.logging.util.RequestLogger;
+
+@Component
+public class LoggingInterceptor implements HandlerInterceptor, RequestLogger {
```

Comment:
once check verification api to see if they are using this in a different way.

Reply:
Checked [identity-verification-api](https://github.com/companieshouse/identity-verification-api/blob/main/src/main/java/uk/gov/companieshouse/identityverificationapi/interceptor/LoggingInterceptor.java
). Ours is the same.